channels/iq.m3u

```diff
@@ -79,6 +79,8 @@ https://linkastream.co/headless?url=https://www.youtube.com/c/karbalaatv/live
 http://163.172.39.215:25461/line/C4@!a3a1@!w72A/58
 #EXTINF:-1 tvg-id="KIRKUKTV.iq" tvg-name="KIRKUK TV" tvg-country="IQ" tvg-language="Kurdish" tvg-logo="" group-title="",KIRKUK TV
 http://163.172.39.215:25461/live/line/C4@!a3a1@!w72A/58.m3u8
+#EXTINF:-1 tvg-id="KorekTV.iq" tvg-name="Korek TV" tvg-country="IQ" tvg-language="Kurdish" tvg-logo="https://i.imgur.com/hGOtVre.png" group-title="",Korek TV
+https://live.karwan.tv/karwan.tv/korek-tv/chunks.m3u8
```

Comment:
There is no sound.

Reply:
Crap! Should've checked sound. I'll try my best to find working streams.